### ISSUE TITLE
Compatibility problem on TicWatch 3 Pro

### ISSUE THREAD

**Issue by fcaronte** [matejdro/WearVibrationCenter#11]
Hi, I discovered this app and I try to install on my phone and TicWatch 3 pro with wear OS 2 but unfortunately seem to do nothing I try t to change settings but I always get stock vibration. I add all permission and remove from battery optimization on my phone. Suggestions?

**Comment by matejdro** [comment]
Can you press "Get support" button at the bottom of the global settings couple of seconds after you receive a notification and sent me the logs?

**Comment by fcaronte** [comment]
[logs.zip](https://github.com/matejdro/WearVibrationCenter/files/7653658/logs.zip)
@matejdro here to you, I read the log and I discovered the problem. Practically notification are get as "persistent notification" and so are filter by the option "only vibrate for notifications that vibrated in the first place" now I try to disable it and I get custom vibration.

**Comment by matejdro** [comment]
So everything is working fine now?

**Comment by fcaronte** [comment]
@matejdro umh i don't think WhatsApp notification should be see as persistent, so there is something strange. If disable the option then I start to receive notifications for weather and other notification 😬

**Comment by matejdro** [comment]
Huh. Can you actually swipe away Whatsapp notification?

**Comment by fcaronte** [comment]
@matejdro yes I can dismiss like normally, is not really a persistent notification is something wrong with detect it, maybe caused by a different MIUI notification management, as now I found I can enable "persistent" notification option as filtered for app and so I enable it for whatsapp and telegram and keep disabled for other app

**Comment by matejdro** [comment]
Yeah it is possible it is something like that. Persistent detection just receives signal from the system, so something weird is going on here.
